packages/nextjs/services/store/tokenStore.ts

```diff
@@ -58,7 +58,6 @@ interface TokenStore {
   loadingTokens: boolean;
 
   pairs: ChainRecord<AddressRecord<ConfidentialTokenPair>>;
-  confidentialToPublicMap: ChainRecord<AddressRecord<Address>>;
```

Comment:
Turns out we weren't using this anywhere. I've removed it because it will cause an issue with ETH/wETH which should both be pointed to by the confidentialeth contract.